.env

```diff
@@ -33,7 +33,7 @@ MANYLINUX=2014
 MAVEN=3.5.4
 PYTHON=3.10
 GO=1.19.5
-ARROW_MAJOR_VERSION=11
+ARROW_MAJOR_VERSION=12
```

Comment:
should bump the go.mod and go.sum by running `go get -u github.com/apache/arrow/go/v12@latest` and then  `go mod tidy`

Reply:
ah, right. also went and bumped the imports